common/beerocks/bwl/dummy/base_wlan_hal_dummy.cpp

```diff
@@ -88,6 +88,7 @@ base_wlan_hal_dummy::base_wlan_hal_dummy(HALType type, std::string iface_name, b
     }
 
     // Initialize the FSM
+    refresh_radio_info();
```

Comment:
Maybe you should keep the logic in other BWL types - refresh_radio_info on attach instead...

Reply:
Done